fulp_modules/features/toys/code/toys.dm

```diff
@@ -24,3 +24,13 @@
 	attack_verb_continuous = list("dust")
 	attack_verb_simple = list("dust")
 	squeak_override = list('sound/effects/supermatter.ogg'= 1)
+
+/obj/item/toy/plush/pico
+	name = "pico plushie"
+	desc = "A plushie of an alarmed middle schooler with orange hair and a green T-Shirt. Maybe something's happening at his school."
```

Comment:
it should be T-shirt or t-shirt

Reply:
Fun fact: It's not a **T** shirt. It's just a short, look at the sleeves.